src/spatial/math.js

```diff
@@ -271,14 +277,13 @@ Crafty.math.Vector2D = (function () {
      * #.add
      * @comp Crafty.math.Vector2D
      * @kind Method
-     * 
+     * @public
      *
      * Adds the passed vector to this vector
      *
-     * @public
      * @sign public {Vector2D} add(Vector2D);
```

Comment:
missing formal parameter name in signature `@sign public {Vector2D} add(Vector2D **vecRH**)`